chain-abci/src/main.rs

```diff
@@ -96,12 +112,19 @@ fn main() {
     let proxy = get_enclave_proxy(&opt);
 
     let addr = SocketAddr::new(opt.host, opt.port);
+
+    let mut app_hash = opt.genesis_app_hash.clone();
+    if opt.genesis_app_hash.len() == 0 {
```

Comment:
You can use `match` or `if let` here if `genesis_app_hash` is `Option<String>`.

Reply:
i'll try with if let